datakyt/db/import/insert_data_in_project_table.py

```diff
@@ -0,0 +1,23 @@
+import csv
+
+
+def insert_csv_file_to_database(path, conn):
+    """
+    importing csv file to a database.
```

Comment:
More details here please: required `csv` file format, what table will be used for insert etc

Reply:
add name columns